packages/cli/src/commands/env/rm.ts

```diff
@@ -0,0 +1,41 @@
+import * as api from '../../rest/api'
+import { Args } from '@oclif/core'
+import { AuthCommand } from '../authCommand'
+
+export default class EnvRm extends AuthCommand {
+  static hidden = false
+  static description = 'Remove environment variable via checkly env rm <key>'
```

Comment:
For consistency with our other commands, all of the descriptions should end with a period.